src/components/NewMessageNotification.js

```diff
@@ -1,34 +1,35 @@
-'use strict';
+'use strict'
 
-import React, { PropTypes } from 'react';
+import React from 'react'
+import PropTypes from 'prop-types'
 
 class NewMessageNotification extends React.Component {
-
-  static propTypes = {
-      onClick: PropTypes.func,
-      unreadMessages: PropTypes.number,
-  };
-
-  render() {
-    const { onClick, unreadMessages } = this.props;
+  render () {
+    const { onClick, unreadMessages } = this.props
     if (unreadMessages > 0) {
       if (unreadMessages === 1) {
         return (
           <div className="newMessagesBar" onClick={onClick}>
             There is <span className="newMessagesNumber">1</span> new message
           </div>
-        );
+        )
       } else {
         return (
           <div className="newMessagesBar" onClick={onClick}>
-            There are <span className="newMessagesNumber">{unreadMessages}</span> new messages
+            There are{' '}
+            <span className="newMessagesNumber">{unreadMessages}</span> new
+            messages
```

Comment:
Something's off on these lines re. the line breaks, hard to read the full structure here.